python/setup.py

```diff
@@ -45,18 +47,19 @@
 
 # Platform-specific adjustments
 if sys.platform == 'win32':
-    ice_cpp_sources.append(f"../cpp/include/generated/{platform}/{configuration}/Ice")
+    ice_cpp_sources.extend([f"../cpp/include/generated/{platform}/{configuration}/Ice",
```

Comment:
This does not look correct.  I believe all the header files go into a local `{platform}/{configuration}/IceDiscovery`.